arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -341,5 +341,58 @@ public async Task<GetGraphEdgeCollectionsResponse> GetGraphEdgeCollectionsAsync(
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Gets an edge from the given graph using the edge collection and _key attribute.
+        /// </summary>
+        /// <typeparam name="T">The type of the edge document to deserialize to.</typeparam>
+        /// <param name="graphName">The name of the graph.</param>
+        /// <param name="collectionName">The name of the edge collection the edge belongs to.</param>
+        /// <param name="edgeKey">The _key attribute of the edge.</param>
+        /// <param name="query"></param>
+        /// <returns></returns>
+        public Task<GetGraphEdgeResponse<T>> GetEdgeAsync<T>(
```

Comment:
Inspired from one of @AlistairB99124 's PR. Providing 2 overloads makes sense to me.